VSRAD.Package/DebugVisualizer/MouseMove/MouseMoveController.cs

```diff
@@ -20,7 +20,7 @@ public MouseMoveController(DataGridView table, TableState state)
             {
                 new ScaleOperation(_table, _state),
                 new PanOperation(_state),
-                new ReorderOperation(_table)
+                new ReorderOperation(_table, _state.ReorderingEnabled)
```

Comment:
I think you can just add an `if` statement here:
```
_operations = new List<IMouseMoveOperation>
{
    new ScaleOperation(_table, _state),
    new PanOperation(_state)
};
if (_state.ReorderingEnabled)
  _operations.Add(new ReorderOperation(_table));
```

This way, you don't need to change anything in the `ReorderOperation` class.